src/views/components/avatarLookup/avatarLookup.tsx

```diff
@@ -140,42 +120,48 @@ export function AvatarLookup() {
     setSelectedOtherAvatarSet(null);
   };
 
-  return (
-    <div>
-      {/*Below is a temporary grid view, to be removed before merge*/}
-      {targetUserOtherApprovedAvatars ? (
-        <div>
-          <h3>
-            <u>{targetUsername}'s other approved avatars:</u>
-          </h3>
-          <div className="grid-container">
-            {targetUserOtherApprovedAvatars.map((avatarSet) => (
-              <div
-                key={avatarSet.avatarSetId}
-                className={`avatarSet grid-item ${
-                  selectedOtherAvatarSet === avatarSet ? 'selected' : ''
-                }`}
-                onClick={() => handleClickOnOtherAvatar(avatarSet)}
-              >
-                <h3 className="avatarTitle">Avatar {avatarSet.avatarSetId}</h3>
-                <img
-                  src={avatarSet.resLink}
-                  alt={`Resistance avatar ${avatarSet.avatarSetId}`}
-                  className="avatarImg"
-                  draggable={false}
-                />
-                <img
-                  src={avatarSet.spyLink}
-                  alt={`Spy avatar ${avatarSet.avatarSetId}`}
-                  className="avatarImg"
-                  draggable={false}
-                />
-              </div>
-            ))}
+  interface AvatarLibraryProps {
+    avatarLibrary: S3AvatarSet[] | null;
+    selectedAvatarSet: S3AvatarSet | null;
+    handleClickOnAvatar: (avatarSet: S3AvatarSet) => void;
+  }
+
+  const AvatarLibraryGridView = ({
+    avatarLibrary,
+    selectedAvatarSet,
+    handleClickOnAvatar,
+  }: AvatarLibraryProps) => {
```

Comment:
Let's pull this out.

Reply:
Let's make a new folder called `avatar` and shove avatarHome and avatarLibrary into it. Make a common folder in there and shove this into there.